script/scriptMaster.sh

```
@@ -184,6 +184,77 @@ scriptMaster() {
   #  error "invalid use case selected ('$SELECTION') - exiting script"
   #  cleanup
   #  exit 1
+  elif [[ "$SELECTION" == "5)"* ]]; then
+    echo "[info] selected: deploy protocol contract to network"
+    local NETWORK=$(cat ./networks | gum filter --placeholder "Network")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)